### ISSUE TITLE
Result rows as array instead of dictionary

### ISSUE THREAD

**Issue by TwoLaid** [SAP/node-hdb#8]
When retrieving rows with the `execute` function, the resulting rows are always dictionaries with column/name pairs. Would it be possible to return a list of arrays in order to maintain the order of columns?

For example, if I execute this query: `SELECT A, B, C FROM MYTABLE`, the resulting rows currently look like this: `{A: 156, B: 78, C: 0}`. However, I think it would be useful to have the option to return rows like this: `[156, 78, 0]`.

**Comment by holgerkoser** [comment]
This will be supported in the next release.
I will work like this
```js
client.exec('select * from DUMMY', {
  rowsAsArray: true
}, function (err, rows) {
  rows.should.eql([
    ['X']
  ]);
});
```

**Comment by holgerkoser** [comment]
New query options `rowsAsArray` is available with version 0.4.2.
